include/fuse_common.h

```diff
@@ -846,6 +847,14 @@ struct fuse_buf {
 	 */
 	void *mem;
 
+	/**
```

Comment:
Would it still break ABI if this was moved to the end of this struct? 

Otherwise, the only other option is to have the server pass in the extended max write size as a command-line arg (see https://github.com/libfuse/libfuse/pull/984), which I'd very much like to avoid if possible.

Reply:
I will take a look tomorrow in the evening, the 4th weekend occupied with something else :(
My guess is that moving to the end would also have some issues, as we don't know the size of the passed struct - don't know if it is initialized, but the good part is that there is `enum fuse_buf_flags flags` - I guess we need to enforce that this flag gets set for the extended struct size. But I really first need to check the code tomorrow.